autobahn/wamp_session.ipp

```diff
@@ -550,6 +557,10 @@ boost::future<wamp_call_result> wamp_session<IStream, OStream>::call(
     msgpack::packer<msgpack::sbuffer> packer(*buffer);
     uint64_t request_id = ++m_request_id;
 
+    /*if (m_caller_disclose_me) {
+        options.set_disclose_me(true);
+    }*/
```

Comment:
@oberstet @jpetso What is the best way to do this given 'options' is a constant?